app/javascript/topics/TopicsShow__Root.res

```diff
@@ -445,8 +455,10 @@ let make = (
             addPostLikeCB={() => send(LikeFirstPost)}
             removePostLikeCB={() => send(RemoveLikeFromFirstPost)}
             markPostAsSolutionCB={() => ()}
+            unmarkPostAsSolutionCB={() => ()}
```

Comment:
If neither of these callbacks _do anything_, then why are they required anymore? 🤔

Reply:
@harigopal: The same component is used for the first post and the replies to the topic. These callbacks are applicable only for the replies. Maybe should consider making this an optional argument in the component?

Reply:
Hmm. Nah... Then there would be the weird case of having to handle an `option` callback in a case where a callback is _definitely_ required. 🤷

Reply:
Tried remembering. Guess that's the reason I took this approach when I got stuck handling this. Will keep it as it is. 👍